Formula/knot-resolver.rb

```diff
@@ -0,0 +1,101 @@
+class KnotResolver < Formula
+  desc "Minimalistic, caching, DNSSEC-validating DNS resolver"
+  homepage "https://www.knot-resolver.cz"
+  url "https://secure.nic.cz/files/knot-resolver/knot-resolver-1.2.1.tar.xz"
+  sha256 "1b6f55ea1dfec90f45c437f23e1ab440e478570498161d0f8a8f94a439305f8c"
+  revision 1
+
+  head do
+    url "https://gitlab.labs.nic.cz/knot/resolver.git"
+  end
+
+  depends_on "knot"
+  depends_on "luajit"
+  depends_on "libuv"
+  depends_on "gnutls"
+  depends_on "lmdb"
+
+  depends_on "cmocka" => :build
+  depends_on "pkg-config" => :run
+
+  option "without-nettle", "Compile without DNS cookies support"
+  option "with-hiredis", "Compile with Redis cache storage support"
+  option "with-libmemcached", "Compile with memcached cache storage support"
+  depends_on "nettle" => :recommended
+  depends_on "hiredis" => :optional
+  depends_on "libmemcached" => :optional
+
+  def install
+    %w[all check lib-install daemon-install modules-install].each do |target|
+      system "make", target, "PREFIX=#{prefix}"
+    end
+
+    (buildpath/"config").write(config)
+    (etc/"kresd").install "config"
+
+    (buildpath/"root.keys").write(root_keys)
+    (var/"kresd").install "root.keys"
+  end
+
+  def config; <<-EOS.undent
```

Comment:
@vcunat Homebrew runs everything as the same user.
@fcelda It would be preferable to `inreplace` an existing example configuration rather than reproducing so much of it in the formula.